ext/FluxnetSimulations.jl

```diff
@@ -0,0 +1,308 @@
+module FluxnetSimulations
```

Comment:
I think a good structure would be to have a main module file but split up the code into different files, like we do for e.g. the Soil module. So the code currently in here would go into something like `ext/FluxnetSimulations.jl/data_cleaning.jl`, and then we include that from this module.

Then Thanhthanh's `get_parameter` and `get_domain_info` functions can go into another file, etc.

Note that even if we do this we're still moving away from the exp/integrated/fluxnet include structure because everything is packaged into functions and in a module :)

Reply:
I split the code up but can do so more if we prefer